Package.swift

```diff
@@ -27,7 +27,7 @@ let package = Package(
                  targets: ["TuistGenerator"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/tuist/XcodeProj", .upToNextMajor(from: "7.5.0")),
```

Comment:
This will need to be updated with a new version after https://github.com/tuist/XcodeProj/pull/521 is merged

Reply:
The changes in that branch looked good. I approved and merged the branch. Once [we merge this](https://github.com/tuist/XcodeProj/pull/519), you can point this to 7.6.0.